src/main/java/com/stripe/param/SubscriptionCreateParams.java

```diff
@@ -0,0 +1,758 @@
+   */
+  @SerializedName("coupon")
+  String coupon;
+
+  /** The identifier of the customer to subscribe. */
+  @SerializedName("customer")
+  String customer;
+
+  /**
+   * Number of days a customer has to pay invoices generated by this subscription. Valid only for
+   * subscriptions where `billing` is set to `send_invoice`.
+   */
+  @SerializedName("days_until_due")
+  Long daysUntilDue;
+
+  /**
+   * ID of the default payment source for the subscription. It must belong to the customer
+   * associated with the subscription and be in a chargeable state. If not set, defaults to the
+   * customer's default source.
+   */
+  @SerializedName("default_source")
+  String defaultSource;
+
+  /** Specifies which fields in the response should be expanded. */
+  @SerializedName("expand")
+  List<String> expand;
+
+  /** List of subscription items, each with an attached plan. */
+  @SerializedName("items")
+  List<Item> items;
+
+  /**
+   * A set of key-value pairs that you can attach to a `Subscription` object. It can be useful for
+   * storing additional information about the subscription in a structured format.
+   */
+  @SerializedName("metadata")
+  Map<String, String> metadata;
+
+  /**
+   * Boolean (defaults to `true`) telling us whether to [credit for unused
+   * time](https://stripe.com/docs/subscriptions/billing-cycle#prorations) when the billing cycle
+   * changes (e.g. when switching plans, resetting `billing_cycle_anchor=now`, or starting a trial),
+   * or if an item's `quantity` changes. If `false`, the anchor period will be free (similar to a
+   * trial) and no proration adjustments will be created.
+   */
+  @SerializedName("prorate")
+  Boolean prorate;
+
+  /**
+   * A non-negative decimal (with at most four decimal places) between 0 and 100. This represents
+   * the percentage of the subscription invoice subtotal that will be calculated and added as tax to
+   * the final amount in each billing period. For example, a plan which charges $10/month with a
+   * `tax_percent` of `20.0` will charge $12 per invoice. To unset a previously-set value, pass an
+   * empty string.
+   */
+  @SerializedName("tax_percent")
+  Object taxPercent;
+
+  /**
+   * If specified, the funds from the subscription's invoices will be transferred to the destination
+   * and the ID of the resulting transfers will be found on the resulting charges. This will be
+   * unset if you POST an empty value.
+   */
+  @SerializedName("transfer_data")
+  TransferData transferData;
+
+  /**
+   * Unix timestamp representing the end of the trial period the customer will get before being
+   * charged for the first time. This will always overwrite any trials that might apply via a
+   * subscribed plan. If set, trial_end will override the default trial period of the plan the
+   * customer is being subscribed to. The special value `now` can be provided to end the customer's
+   * trial immediately.
+   */
+  @SerializedName("trial_end")
+  String trialEnd;
```

Comment:
This should be a string for `now` or a `long` for unix timestamp. Would it be cleaner to handle this the right way?

Reply:
below here too